addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1257,6 +1257,15 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * @override
+     */
+    open: async function () {
+        const _super = this._super.bind(this);
+        await this._renderColorPalette();
+        _super(...arguments);
+    },
```

Comment:
This commit is not needed anymore since of your new work, remove it or squash it